projects/sandbox/vizapp/vizapp/plots/background.py

```diff
@@ -83,9 +87,9 @@ def configure_plots(self, height: int, width: int):
         )
         self.distribution_plot.add_layout(axis, "right")
 
-        r = self.distribution_plot.circle(
-            "detection_statistic",
-            "snr",
+        self.injection_renderer = self.distribution_plot.circle(

```

Comment:
Small thing, but is there a reason this is saved as an attribute? Just curious

Reply:
Think this is just my bad habit of saving everything as attributes